Podfile

```diff
@@ -222,7 +222,7 @@ abstract_target 'Apps' do
 
   pod 'Gridicons', '~> 1.1.0'
 
-  pod 'WordPressAuthenticator', '~> 2.1'
+  pod 'WordPressAuthenticator', '~> 2.3.0'
```

Comment:
This is the rule necessary to not automatically to `2.4.0`, which introduced a breaking change.

See
https://buildkite.com/automattic/wordpress-ios/builds/9808#01830b2b-76f3-4fe1-b385-52e09f417c3c:

```
WordPress/Classes/ViewRelated/NUX/WordPressAuthenticationManager.swift:228:1: type 'WordPressAuthenticationManager' does not conform to protocol 'WordPressAuthenticatorDelegate'
```